fetch-ledger-develop-image/Dockerfile

```diff
@@ -30,6 +32,7 @@ RUN HOME=/home/default && \
 
 ENV CC=/usr/bin/clang
 ENV CXX=/usr/bin/clang++
+ENV CCACHE_DIR=/build/docker_ccache
```

Comment:
This is useful for me to have persistent caching. Note ccache is not installed in the container by default